outputs.tf

```diff
@@ -52,6 +52,20 @@ output "orca_lambda_post_copy_request_to_queue_arn" {
   value       = module.orca.orca_lambda_post_copy_request_to_queue_arn
 }
 
+output "orca_catalog_reporting_api_invoke_url" {
```

Comment:
See comments from modules/lambdas/main.tf. I am thinking we should just have a single base URL instead of three.